bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -50,8 +50,9 @@ class GenerateAddressesOptions():
         options: An instance of `GenerateAddressOptions`.
     """
 
+    # pylint: disable=redefined-builtin
     def __init__(self, coinType: CoinType,
-                 range: range,
+                 range: Range,
```

Comment:
What was `range`? Is that a default Python type? If yes, then this is a breaking change?

Reply:
range is a std method.